src/borg/testsuite/archiver.py

```diff
@@ -373,14 +380,21 @@ def filter(output):
 
     def test_unix_socket(self):
         self.cmd('init', self.repository_location)
-        sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
-        sock.bind(os.path.join(self.input_path, 'unix-socket'))
-        self.cmd('create', self.repository_location + '::test', 'input')
-        sock.close()
-        with changedir('output'):
-            self.cmd('extract', self.repository_location + '::test')
-            assert not os.path.exists('input/unix-socket')
-
+        try:
+            sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+            sock.bind(os.path.join(self.input_path, 'unix-socket'))
+            self.cmd('create', self.repository_location + '::test', 'input')
+            sock.close()
+            with changedir('output'):
+                self.cmd('extract', self.repository_location + '::test')
+                assert not os.path.exists('input/unix-socket')
+        except PermissionError as err:
```

Comment:
The reason I put it here is so that if some operation borg performs on the sockets isn't supported, it'll get caught. It's only catching `PermissionError`s after all, those are very unlikely to actually be borg's fault.